assets/scss/_bootstrap_custom.scss

```diff
@@ -14,5 +14,6 @@
 
 .card-mindwendel {
   @extend .card;
-  width: 285px;
+  width: 286px;
```

Comment:
I think we'll probably need to start working with % here at some point, not sure how far `px` can carry us here but I'm also not great at CSS

Reply:
yes good point, updated it with `calc` as we do need to include card margins. Also fixed it for mobile.